package com.tcs.exceptions;

//an exception is an object
//objects are created at runtime means an exception will occur only at runtime
//when it occurs the program ,, stops abruptly if it is not handled

public class TestExceptions {
	
	public static void main(String[] args) {
		System.out.println(10/0);
		System.out.println("after division");
	}
	
	

}
